pinot-core/src/test/java/org/apache/pinot/queries/FilteredAggregationsTest.java

```diff
@@ -296,6 +296,19 @@ public void testMultipleAggregationsOnSameFilter() {
         + "MAX(CASE WHEN (INT_COL > 29990) THEN INT_COL ELSE 0 END) AS total_max, "
         + "SUM(CASE WHEN (NO_INDEX_COL < 5000) THEN INT_COL ELSE 0 END) AS total_sum, "
         + "MAX(CASE WHEN (NO_INDEX_COL < 5000) THEN NO_INDEX_COL ELSE 0 END) AS total_max2 FROM MyTable";
+  }
+
+  @Test
+  public void testMixedAggregations() {
```

Comment:
Lets rename this to testMixedAggregationsOfSameType, since other tests in this class contain tests with mixed aggregations, just of different types

Reply:
Done